ddl/reorg.go

```diff
@@ -161,6 +165,37 @@ func (w *worker) runReorgJob(t *meta.Meta, reorgInfo *reorgInfo, lease time.Dura
 	}
 }
 
+func getTableTotalCount(w *worker, tblInfo *model.TableInfo) int64 {
```

Comment:
Can we just `select table_rows from infomation_schema.tables`? You don't even need to handle partitions.

Reply:
great.